src/features/projects/components/maps/Explore.tsx

```diff
@@ -3,8 +3,7 @@ import styles from '../../styles/ProjectsMap.module.scss';
 import CancelIcon from '../../../../../public/assets/images/icons/CancelIcon';
 import ExploreIcon from '../../../../../public/assets/images/icons/ExploreIcon';
 import i18next from '../../../../../i18n';
-import FormControlLabel from '@material-ui/core/FormControlLabel';
-import FormGroup from '@material-ui/core/FormGroup';
+import { Modal, FormGroup, FormControlLabel } from '@mui/material';
```

Comment:
There are minor styling changes seen in the Modal. 
![Screenshot 2022-05-05 at 4 28 59 PM](https://user-images.githubusercontent.com/44917347/166910147-d1048061-22b3-41dd-8a65-6c5558a0cdc0.png)

Reply:
Fixed in #1467